### DISCUSSION TITLE
Idiomatic way to add style to innerHTML

### DISCUSSION THREAD

**Discussion by kyoheiu** [tailwindlabs/tailwindcss#11992] [Help] [answered]
My question is simple: What is the idiomatic way in tailwindcss to add style to innerHTML, which we cannot set style explicitly, when it is not recommended to use `@apply` in css file?
cf: [Reusing Styles - Tailwind CSS](https://tailwindcss.com/docs/reusing-styles#avoiding-premature-abstraction)

**Comment by wongjn** [comment, marked as answer]
If the inner content is prose-like content, consider using the [`@tailwindcss/typography` official Tailwind plugin](https://tailwindcss.com/docs/typography-plugin). Otherwise, consider using [arbitrary variants](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-variants).

**Reply by kyoheiu** [reply]
Gotcha, thank you!

I think it's worth mentioning in the page above, because when migrating from @apply people might be confused by edge cases such as innerHTML.
